numpy/lib/function_base.py

```diff
@@ -1332,6 +1332,10 @@ def interp(x, xp, fp, left=None, right=None, period=None):
         If `xp` or `fp` are not 1-D sequences
         If `period == 0`
 
+    See also
```

Comment:
```suggestion
    See Also
```
per the [numpydoc standard](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard).